ios-release-build.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: page
+title: Preparing an iOS App for Release
+sidebar: home_sidebar
+permalink: /iOS-release/
+---
+
+During the typical development cycle, you'll test an app using `flutter run` at
+the command line, the Run and Debug toolbar buttons in IntelliJ, or both. By
+default, Flutter builds a *debug* version of your app.
+
+When you're ready to prepare a *release* version, for example to [publish to
+the App Store][appstore], follow the steps on this page. To release an app to
+the App Store, you will need a registered [Apple Developer Program][devprogram]
+account.
+
+This guide provides a step-by-step walkthrough of releasing a typical app to
+the App Store. For a detailed overview, see Apple's [App Distribution
+Guide][distributionguide].
+
+*Note*: Many of the steps below must be carried out in Xcode. To open your app
+in Xcode:
+
+  * From Xcode: Invoke `File > Open` and select the `ios` folder located inside
+    your main app directory.
+
+  * From the terminal: `cd <app dir>; open ios/Runner.xcworkspace`.
+
+* TOC Placeholder
+{:toc}
+
+
+## Register your app on iTunes Connect
```

Comment:
Maybe move this section down to after `## Signing the app` to make it flow more chronologically?